spec/models/measure_spec.rb

```diff
@@ -251,4 +251,22 @@
       it { expect(measure.excluded_country_list).to eq(expected_list) }
     end
   end
+
+  describe '#measure_conditions_with_valid_guidance' do
+    context 'with a valid condition guidance' do
+      let(:measure) { build :measure, :with_conditions_with_guidance }
+
+      it 'returns measure_conditions with valid guidance' do
```

Comment:
```suggestion
      it 'returns measure_conditions with guidance' do
```

I think we know we have/don't have guidance. I'm not sure this is valid/invalid test, etc